packages/url_launcher/url_launcher/example/integration_test/url_launcher_test.dart

```diff
@@ -17,6 +17,7 @@ void main() {
 
     // Generally all devices should have some default browser.
     expect(await canLaunch('http://flutter.dev'), true);
+    expect(await canLaunch('https:///www.google.com/404'), true);
```

Comment:
Why is there an extra `/` after the scheme here?

Reply:
Good catch. done